application-licensing-licensor/application-licensing-licensor-api/src/main/resources/templates/extensionAutoUpgraded/extensionAutoUpgraded.vm

```diff
@@ -0,0 +1,53 @@
+#template('notification/macros.vm')
+
+#includeMacros('Licenses.Code.VelocityMacros')
+
+#set ($mainIcon = "bell")
+#set ($smallIcon = "arrow_up")
+#set ($compositeEventDate = $escapetool.xml($datetool.whenIs($event.dates.get(0))))
+
+#macro (displayEventDetails $event)
+  #set ($userName = $event.user.name)
+  <tr>
+    <td>
+      <span class="notification-event-user">$userName</span>
```

Comment:
If it doesn't need to be XML escaped then I would add a comment explaining why (does it contain a link to the user profile?).